### PR TITLE
[Storage][Blob]Regenerate Swagger Code

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py [hunk 1]
```diff
@@ -167,6 +167,7 @@ class StorageErrorCode(str, Enum):
     lease_not_present_with_container_operation = "LeaseNotPresentWithContainerOperation"
     lease_not_present_with_lease_operation = "LeaseNotPresentWithLeaseOperation"
     max_blob_size_condition_not_met = "MaxBlobSizeConditionNotMet"
+    no_authentication_information = "NoAuthenticationInformation"
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py]: I think this code needs to be added to the Error code enum in the `_shared` directory:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py#L19

File: sdk/storage/azure-storage-blob/azure/storage/blob/_models.py [hunk 2]
```diff
@@ -1079,7 +1079,7 @@ def _from_generated(cls, generated):
         if generated.properties.default_encryption_scope:
             scope = cls(
                 generated.properties.default_encryption_scope,
-                prevent_encryption_scope_override=generated.properties.deny_encryption_scope_override or False
+                prevent_encryption_scope_override=generated.properties.prevent_encryption_scope_override or False
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_models.py]: Does this header need updating as well? Or does that remain the same?
https://github.com/Azure/azure-sdk-for-python/blob/bea1bea93e2e432052432eedd07068f6fe66fd7c/sdk/storage/azure-storage-blob/azure/storage/blob/_models.py#L333

Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_models.py]: Good question! The service header will not be updated

**Pull request merged**